kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue

```diff
@@ -421,11 +419,14 @@
         );
         this.progressStartingPoint = this.dummyTime;
       },
+      // An alternative to recordProgress, this updates tracking based on clock-time spent on media
+      updateProgress() {
```

Comment:
Let's revert this for now, and just stick with recordProgress, as messing with this all seemed a bit more complex than we anticipated.

Reply:
Hmm, does this mean that media player won't be using `defaultDuration`, since it doesn't use `forceDurationBasedProgress`, and thus `updateProgress()`?

Essentially, just go back to what it originally was?